client/src/reducers/chatReducer.js

```diff
@@ -0,0 +1,80 @@
+const ADD_MESSAGE = 'ADD_MESSAGE';
+const SET_CURRENT_PERSON = 'SET_CURRENT_PERSON';
+const CLEAR_CURRENT_PERSON = 'CLEAR_CURRENT_PERSON';
+let index;
+
+const defaultState = {
+  chats: [
+    {
+      id: 2,
+      person: 'Bob',
+      messages: [
+        ['Здравствуйте', '27.09.2021, 08:03:09'],
```

Comment:
Chat messages doesn't contain the flag of whose message is this (yours or other person's)